substrate/frame/revive/src/limits.rs

```diff
@@ -73,6 +73,14 @@ pub const PAGE_SIZE: u32 = 4 * 1024;
 /// Which should always be enough because Solidity allows for 16 local (stack) variables.
 pub const IMMUTABLE_BYTES: u32 = 4 * 1024;
 
+/// upperbound of memory that can be used by the EVM interpreter.
+pub const EVM_MEMORY_LIMIT: u32 = 1024 * 1024;
+
+/// EVM interpreter stack limit.
+pub const EVM_STACK_LIMIT: u32 = 1024;
+pub const EVM_MAX_CODE_SIZE: u32 = 0x6000;
+pub const EVM_MAX_INITCODE_SIZE: u32 = 2 * EVM_MAX_CODE_SIZE;
```

Comment:
let's use `revm::primitives::eip3860::MAX_INITCODE_SIZE` since they are aleady defined in revm and used in vm/evm.rs